concourse/steps/scan_sources.mako

```diff
@@ -29,12 +29,16 @@ scan_sources_and_notify(
 % endif
 
 % if whitesource_cfg:
+component_name = "${component_trait.component_name()}"
 scan_component_with_whitesource(
     whitesource_cfg_name='${whitesource_cfg.cfg_name()}',
     product_token='${whitesource_cfg.product_token()}',
     component_descriptor_path=component_descriptor_path(),
     extra_whitesource_config={},
     requester_mail='${email_recipients[0]}',
+    cve_threshold=,'${whitesource_cfg.cve_threshold()}',
```

Comment:
nit: should not this rather be a float/int?
also, I believe there is a syntax error (comma after equals sign)